daml_dit_ddit/common.py

```diff
@@ -18,7 +18,7 @@
 
 from .log import LOG
 
-
+EXPERIMENTAL_NAME = 'experimental'
```

Comment:
Ideally, this would be in the `daml_dit_api` package, but not necessary for this PR. However, do change it to `EXPERIMENTAL_TAG` for consistency with this:

https://github.com/DACH-NY/daml-cloud/blob/master/arcade-repo/arcade_repo/app/package_loader.py#L14